lib/checkPullRequestLabels.js

```diff
@@ -5,6 +5,7 @@ const {
 } = require('../userWhitelist.json');
 const PR_LABELS = ['dependencies',  'stale'];
 const DEFAULT_CHANGELOG_LABEL = 'PR CHANGELOG: Miscellaneous -- @ankita240796';
+const EXCLUDED_LABELS = ['PR CHANGELOG: Angular Migration']
```

Comment:
Make the name more descriptive - from what are the labels excluded, you can add a comment either.

Reply:
Done.